tasks/coverage.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :coverage do
+  desc 'Report Coverage from merged CI runs'
+  task :ci do
+    require 'simplecov'
+
+    SimpleCov.collate Dir['coverage-*/.resultset.json']
```

Comment:
Will this combine results from different Ruby versions, and Parser results with Prism results?

Reply:
Yes. Different ruby versions aren't really needed since it will be at 100% with just two jobs right now (one prism, one parser) but limiting it in the workflow adds complexity which I don't think is really needed.

Just doing it will all runs seems ok to me for simplicity.